Source/WebCore/page/DragController.cpp

```diff
@@ -162,7 +162,7 @@ static RefPtr<DocumentFragment> documentFragmentFromDragData(const DragData& dra
             if (!url.isEmpty()) {
                 auto& document = context.start.document();
                 auto anchor = HTMLAnchorElement::create(document);
-                anchor->setHref(url);
+                anchor->setHref(AtomString { url });
```

Comment:
Does the `DragData::asURL` function need to return `String` instead of `AtomString` to keep other code paths efficient?

Reply:
Yes, it is n url so not a particularly great fit for AtomString, also, asURL() is called from quite a few places, most of them just needing a String (e.g. to construct a URL from it).